modules/integration/tests-common/admin-clients/src/main/java/org/wso2/appserver/integration/common/clients/EventProcessorAdminServiceClient.java

```diff
@@ -125,4 +136,51 @@ public void removeInactiveExecutionPlan(String filePath)
             throw new RemoteException();
```

Comment:
@thiliA 
Same comment as above. There seems to be multiple places where the original exception has been dropped. Can you please fix those in all places that this issue occurs?